subsys/nrf_security/src/drivers/cracen/cracenpsa/src/cipher.c

```diff
@@ -141,28 +141,29 @@ static psa_status_t crypt_ecb(const struct sxkeyref *key, const uint8_t *input,
 	}
 
 	*output_length = 0;
+	memset(blkciph, 0x0, sizeof(struct sxblkcipher));
```

Comment:
`safe_memzero` instead?

Reply:
Oupsi, yes,  will do it for consistency!